src/_cffi_src/openssl/ssl.py

```diff
@@ -582,6 +590,12 @@
 #if CRYPTOGRAPHY_IS_LIBRESSL
 static const long SSL_OP_NO_DTLSv1 = 0;
 static const long SSL_OP_NO_DTLSv1_2 = 0;
+/* it's not really clear why DTLS1_VERSION is defined, but it is. */
```

Comment:
What's `DTLS1_VERSION`? I don't see that anywhere in the code.

Reply:
Defined above, it's one of the new OpenSSL version constants for DTLS. 

LibreSSL doesn't support DTLS 1.2 (https://github.com/libressl-portable/portable/issues/380), but it's a bit of a puzzle to me whether/how they do support DTLS 1.0. 

At the very least, they have DTLS1_VERSION defined (but apparently not SSL_OP_NO_DTLSv1 if I understand line 591 correctly?), and setting that to =0 here (as I do with DTLS1_2_VERSION ) would break the build.

Reply:
If there's no need for the constant, just don't expose it :-)